### ISSUE TITLE
Highlights stick around after item dissapears

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/john-travoltage#407]
**Test device**
Dell
**Operating System**
Win 10
**Browser**
Firefox
**Problem description**
For https://github.com/phetsims/QA/issues/645.
When an item is highlighted, and clicking on it makes the item go away (such as the x in a dialog) the highlight sticks around until the mouse is move again. Does not happen with keyboard nav.
**Steps to reproduce**
1. Go to Preferences Visual tab and turn on highlighting
2. Click on the x to leave the dialog

**Visuals**
![highlightsticksaround2](https://user-images.githubusercontent.com/41024075/117706336-69966800-b18a-11eb-8bb5-10718bd20621.gif)
![highlightsticksaround](https://user-images.githubusercontent.com/41024075/117706350-6c915880-b18a-11eb-80f3-4c34337c9e24.gif)

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪John Travoltage‬
URL: https://phet-dev.colorado.edu/html/john-travoltage/1.6.0-preferences.4/phet/john-travoltage_en_phet.html
Version: 1.6.0-preferences.4 2021-04-28 22:49:44 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:88.0) Gecko/20100101 Firefox/88.0
Language: en-US
Window: 1280x687
Pixel Ratio: 1.5/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0
Vendor: Mozilla (Mozilla)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by KatieWoe** [comment]
It seems worse on iPad. Some of the highlights also don't show up on ipad, such as on the reset all button.

https://user-images.githubusercontent.com/41024075/117729205-c2c0c480-b1a7-11eb-90c2-b608db3ff076.MP4

**Comment by jessegreenberg** [comment]
This is caused by the same problem in #412 and has been resolved as part of that issue.
